### ISSUE TITLE
[sbs] HTTPError: 403 Client Error: Forbidden for url: http://www.sbs.com.au/api/video_menu/?group=41&context=android&form=json

### ISSUE THREAD

**Issue by aussieaddonsissues** [aussieaddons/issue-reports#5784]
*Automatic bug report from end-user.*

## Environment

**Add-on Name:** SBS On Demand
**Add-on ID:** plugin.video.sbs
**Add-on Version:** 1.1.3-4~g0324247
**Add-on URL:** 
**Kodi Version:** 18.6
**Python Version:** 2.7.17 (default, Feb 26 2020, 22:32:04) [GCC 9.2.0]
**IP Address:** 101.116.239.171
**Hostname:** N/A
**Country:** AU
**ISP:** AS133612 Vodafone Australia Pty Ltd
**Operating System:** linux2  (Linux 5.4.0-gentoo x86_64)
**Platform:** Linux
**Python Path:**
```
/home/rick/.kodi/addons/plugin.video.sbs/resources/lib
plugin://plugin.video.sbs
/home/rick/.kodi/addons/plugin.video.sbs
/home/rick/.kodi/addons/script.common.plugin.cache/resources/lib/storage_server/
/home/rick/.kodi/addons/script.module.aussieaddonscommon/lib
/home/rick/.kodi/addons/script.module.beautifulsoup4/lib
/home/rick/.kodi/addons/script.module.certifi/lib
/home/rick/.kodi/addons/script.module.chardet/lib
/home/rick/.kodi/addons/script.module.future/libs
/home/rick/.kodi/addons/script.module.idna/lib
/home/rick/.kodi/addons/script.module.requests/lib
/home/rick/.kodi/addons/script.module.urllib3/lib
/usr/lib64/python27.zip
/usr/lib64/python2.7
/usr/lib64/python2.7/plat-linux2
/usr/lib64/python2.7/lib-tk
/usr/lib64/python2.7/lib-old
/usr/lib64/python2.7/lib-dynload
/usr/lib64/python2.7/site-packages
/usr/lib64/python2.7/site-packages/gtk-2.0
/home/rick/.kodi/addons/script.module.future/libs/future/standard_library/Lib
```

## Traceback
```
Traceback (most recent call last):
  File "/home/rick/.kodi/addons/plugin.video.sbs/resources/lib/index.py", line 29, in make_index_list
    index = comm.get_index()
  File "/home/rick/.kodi/addons/plugin.video.sbs/resources/lib/comm.py", line 104, in get_index
    resp = fetch_cache_url(config.index_url)
  File "/home/rick/.kodi/addons/plugin.video.sbs/resources/lib/comm.py", line 56, in fetch_cache_url
    return cache.cacheFunction(fetch_url, url)
  File "/home/rick/.kodi/addons/script.common.plugin.cache/resources/lib/storage_server/StorageServer.py", line 623, in cacheFunction
    ret_val = funct(*args)
  File "/home/rick/.kodi/addons/plugin.video.sbs/resources/lib/comm.py", line 44, in fetch_url
    request = sess.get(url)
  File "/home/rick/.kodi/addons/script.module.requests/lib/requests/sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)
  File "/home/rick/.kodi/addons/script.module.aussieaddonscommon/lib/aussieaddonscommon/session.py", line 58, in request
    raise e
HTTPError: 403 Client Error: Forbidden for url: http://www.sbs.com.au/api/video_menu/?group=41&context=android&form=json

```

[Full log](https://gist.github.com/961c5422b2800e1dab468a4c1b9eede9)

**Comment by shiznix** [comment]
Seems to have been a change at SBS' end that they have since resolved, apologies for the noise.

I'd close this, but can only comment.

**Comment by glennguy** [comment]
hey @shiznix 
It's all good, we probably only get a comment from the user on 1 out of every 2 or 3 hundred of these issue reports, thanks for checking in!

The SBS addon is due for an overhaul, if only to make it compatible with Kodi 19. The current addon uses a seemingly deprecated API and has quite a few of these intermittent 403 issues. I'm hoping moving to the new API will fix that, along with any missing content.

**Comment by shiznix** [comment]
Hi @glennguy 
No problem.

The addons are very nicely done, use them so much and for so long now I'm still privately maintaining the removed TenPlay addon at https://github.com/shiznix/plugin.video.catchuptv.au.ten

Unsure of the politics that went down with that, but I figure as long as it's not advertised or made available in an installable repo for Kodi, then 'should' be all good.

Thanks for replying :)

**Comment by glennguy** [comment]
That's the nature of open source, it would be very hard to 'delete' it from the internet.

What happened was the usual takedown notice, citing copyright infringement and alleging that aussieaddons.com were representing having a commercial relationship with them and making their content 'available' through the software. Obviously the logo is a big culprit, and it seemed apparent that they must think the content is being pirated somehow rather than the reality of Kodi and the addon being a somewhat customised browser tuned specifically for their website. If I forked Chrome and customised how it displayed their website it would in practice be no different.

Really I think the main issue was the aussie addons website, I can see how the reasonable person may conceivably have become confused as to the affiliation of the software. So the website was changed and moving forward the logos are being replaced with original artwork.

I guess if you have any issues you'll hear from GitHub!
